std/utf.d

```diff
@@ -3110,7 +3110,8 @@ enum dchar replacementDchar = '\uFFFD';
  * Params:
  *      r = input range of characters, or array of characters
  * Returns:
- *      input range
+ *      An input range with a length if `isAggregateType!R` is `true`.
+ *      Otherwise, this returns a random access range with slicing.
```

Comment:
The line you linked says it could be infinite instead. Yes, a range over a string is not going to be infinite, but I don't think all this is obvious enough. Especially when the first sentence mentions length, but the second one doesn't.